tests/models/wko/src/example.py

```diff
@@ -167,8 +170,16 @@ def path_goals(self):
         """
         goals = super().path_goals().copy()
 
-        for s in self.energy_system_components["heat_source"]:
-            goals.append(MinimizeSourcesHeatGoal(s))
+        try:
+            self.energy_system_components["heat_source"]
+
+            for s in self.energy_system_components["heat_source"]:
+                goals.append(MinimizeSourcesHeatGoal(s))
+        except KeyError:
+            ...
+        except Exception:
+            traceback.print_exc()
+            sys.exit(1)
```

Comment:
what is the reason for this try except statement? maybe add a comment.

Reply:
Thanks for noting this. I think I added this due to my network not having a heat_source (heat pump) in starting testing. It only had an ATES with a cold and hot demand. I realized afterwards that this does not really make sense, because then the heat and cold demand must be in balance which is not realistic. I have deleted this "try....